.gitlab/deps_build.yml

```diff
@@ -44,6 +44,8 @@ build_libbcc_arm64:
       allow_failure: true
   stage: deps_build
   script:
+    # remove git remote to prevent capture by clang build
+    - git remote remove origin
```

Comment:
Can we do all of this in a temp folder?

Reply:
Maybe. Checking with folks to see if that escapes the CI repo.

Reply:
Done.